include/rmm/mr/device/thrust_allocator_adaptor.hpp

```diff
@@ -50,80 +50,83 @@ class thrust_allocator : public thrust::device_malloc_allocator<T> {
    */
   template <typename U>
   struct rebind {
-    using other = thrust_allocator<U>;
-  };
+    using other = thrust_allocator<U>;  ///< The type to bind to
```

Comment:
This diff looks very different. It looks like everything below here was moved into the definition of `rebind` rather than the definition of `thrust_allocator`? Missed `}`?

Reply:
Yes, I did :) I fixed it locally, but thanks for the find.